### ISSUE TITLE
"Monster shapes", anomalies in dataset.

### ISSUE THREAD

**Issue by sebftw** [gulvarol/surreal#31]
In the README you mention the anomalies you have detected in the dataset `18`, `19`, `143_21`.
I was wondering if you have a more exhaustive list? Or a list of those that are certainly correctly detected. It would be very useful to me!

Also, is the framerate the same for all clips? E.g.  `106_XX` seems very fast compared to `01_XX`.

**Comment by gulvarol** [comment]
Monsters: Unfortunately, I don't have an exhaustive list. I just treated those sequences as noise and included everything in training.

Frame rate: The CMU MoCap has mostly 120 as the frame rate, but there are some sequences with 60 ([example](http://mocap.cs.cmu.edu/search.php?subjectnumber=89)). We downsample all with a factor of 4. See the stepsize parameter [here](https://github.com/gulvarol/surreal/blob/master/datageneration/config.copy#L15). The frame rate information is given on their webpage, but I have realized sometimes it might be wrong, for example it says 120 for Subject 106: http://mocap.cs.cmu.edu/search.php?subjectnumber=106
